presto-native-execution/presto_cpp/main/http/HttpClient.cpp

```diff
@@ -246,6 +246,21 @@ class ConnectionHandler : public proxygen::HTTPConnector::Callback {
   }
 
   void connectSuccess(proxygen::HTTPUpstreamSession* session) override {
+    auto httpInitialReceiveWindow =
```

Comment:
Shall we get these configs on http client constructor and pass to ConnectionHandler on constructor? Then we can avoid accessing the system config in an async execution env? Also could you help also check if we can only set some of the three parameters but not all? Thanks!